src/types/message.ts

```diff
@@ -280,10 +286,10 @@ export function isBadMessage(message: Message): boolean {
 /**
  * All types of messages sent to and from the GCS.
  */
-export type Message = StartMessage | AddMissionMessage | PauseMessage | ResumeMessage | StopMessage
-| ConnectionAckMessage | UpdateMessage | POIMessage | CompleteMessage | ConnectMessage
-| AcknowledgementMessage | BadMessage;
 
+export type Message = StartMessage | AddMissionMessage | PauseMessage | ResumeMessage | StopMessage
+| GeofenceMessage | ConnectionAckMessage | UpdateMessage | POIMessage | CompleteMessage
+| ConnectMessage | AcknowledgementMessage | BadMessage;
 /**
```

Comment:
my bad, lets put the extra line back between line 292 and 293, and remove the unnecessary line in 289